docs/DEVELOPMENT.md

```diff
@@ -116,7 +118,55 @@ Then after this you can:
 
 You may have to run these commands as root/superuser.
 
----
+#### Running `docker-compose up` results in incorrect file system permissions on `/code/node_modules` directory 
+
+While running `docker-compose up`, on Ubuntu, Fedora or other Linux-based systems, you may observe the following error message: 
+
+```
+web      | [BE] $ cd .. && node --trace-warnings server/js/main.js
+web      | [BE] Could not load config.json, using defaults (error message: ENOENT: no such file or directory, open './config.json')
+web      | [BE] unhandled promise rejection TypeError: Cannot read properties of null (reading 'includes')
+web      | [BE]     at resolveWellKnownUri (/code/node_modules/openid-client/lib/issuer.js:179:23)
+web      | [BE]     at Function.discover (/code/node_modules/openid-client/lib/issuer.js:142:26)
+web      | [BE]     at setupAuthRouter (/code/server/src/auth-router.ts:36:34)
+web      | [BE]     at Server.setupApp (/code/server/src/server.ts:125:36)
+web      | [BE]     at runServer (/code/server/src/main.ts:27:18)
+web      | [BE]     at Object.<anonymous> (/code/server/src/main.ts:34:1)
+web      | [BE]     at Module._compile (node:internal/modules/cjs/loader:1364:14)
+web      | [BE]     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1422:10)
+web      | [BE]     at Module.load (node:internal/modules/cjs/loader:1203:32)
+web      | [BE]     at Function.Module._load (node:internal/modules/cjs/loader:1019:12)
+web      | [BE]     at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:128:12)
```

Comment:
Hey @KathyReid , that's not an error that is related to wrong user permissions. That's just a residue from old times when some devs created their own `config.json` to overwrite config parameters. The right error message is something in the lines of
```
EACCES permission denied mkdir /code/node_modules
```
If possible it would be great to put the right error message here or else just remove this one, as it has nothing to do with the permissions :)

Reply:
Good call, I have removed this block and added examples of two errors observed when the permission issue occurs.